card/mod/core/set/all/name_events.rb

```diff
@@ -82,7 +82,7 @@ def changing_existing_tag_to_junction?
 end
 
 def name_in_use_as_tag?
-  Auth.as_bot { Card.count_by_wql right_id: id }.positive?
+  !Card.where(right_id: id, trash: false).take.nil?
```

Comment:
Not sure this even gets used.  I don't think `changing_name_to_junction?` is ever true.

But if it _were_ used, we don't need to count them all!